types/react-native/index.d.ts

```diff
@@ -1,6 +1,6 @@
 // Type definitions for react-native 0.44
 // Project: https://github.com/facebook/react-native
-// Definitions by: Eloy Durán <https://github.com/alloy>, Fedor Nezhivoi <https://github.com/gyzerok>
+// Definitions by: Eloy Durán <https://github.com/alloy>, Fedor Nezhivoi <https://github.com/gyzerok>, HuHuanming <https://github.com/huhuanming>
```

Comment:
I have recently added @types/react-native-google-analytics-bridge, @types/react-navigation, @types/jpush-react-native and some improves on other types repo.
I hope that I can make my contribution to the TypeScript and Developers.